WordPress/Tests/Theme/NonPrintableCheckUnitTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Unit test class for WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Unit test class for the NonPrintableCheck sniff.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @since   0.xx.0
+ */
+class WordPress_Tests_Theme_NonPrintableCheckUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Returns the lines where errors should occur.
+	 *
+	 * @return array <int line number> => <int number of errors>
+	 */
+	public function getErrorList() {
+		return array();
+	}
+
+	/**
+	 * Returns the lines where warnings should occur.
+	 *
+	 * @param string $testFile The name of the file being tested.
+	 *
+	 * @return array <int line number> => <int number of warnings>
+	 */
+	public function getWarningList( $testFile = '' ) {
+		switch ( $testFile ) {
+			case 'NonPrintableCheckUnitTest.css':
+				return array(
+					3 => 1,
+					5 => 1,
+					6 => 1,
+					7 => 1,
+				);
+				break;
```

Comment:
No need to `break` if you're `return`ing in the line above as that effectively breaks you out of the switch (and the whole function) already. (same below)